yal/rapidpro.py

```diff
@@ -187,3 +187,31 @@ async def get_global_flag(global_name):
                 else:
                     continue
     return is_active
+
+
+async def get_rapidpro_global(global_name):
```

Comment:
We're already in the rapidpro class so the name of this function can be something like `get_global_value`